packages/geocoder/src/apis/here/index.ts

```diff
@@ -42,6 +42,7 @@ function run({ options, query, url }: HereFetchArgs): Promise<HereResponse> {
 }
 
 const checkItemInBoundary = ({ rect }: Boundary) => ({ position }: Item) => {
+  if (!position) return true // Chain queries might not have position, ignore them
```

Comment:
This may be me not fully understanding how HERE responses work, but why do we mark unknown position queries as being in the bounding box? Is this to avoid a situation with 0 results returned?

Reply:
There are those weird chain query responses, and in most cases they get removed in a later stage. It doesn't really matter if this is true or false, since they'll get removed later, but I set it to true just to be consistent in case we want to use them at some point in the future.

Reply:
Sounds good! Agreed better to keep them then to remove them